backend/src/data/migrations/20220823095835_seed_test_users_groups.ts

```diff
@@ -0,0 +1,156 @@
+import { genSalt as genPasswordSalt, hash as hashPassword } from 'bcrypt';
```

Comment:
this isn't part of the business logic, so we need to use `seeding` ORM functionality